src/PHPUnit/CommandBuilder/CommandBuilder.ts

```diff
@@ -30,6 +31,12 @@ export class CommandBuilder {
         return this;
     }
 
+    setExtraArguments(extraArguments: string[]) {
```

Comment:
maybe line 22, setArguments change to addArguments, we don't need this method. I'll refactor this later